docker-compose.yml

```diff
@@ -203,4 +203,5 @@ services:
       dockerfile: ./packages/documents/Dockerfile
     restart: unless-stopped
     environment:
+      - NODE_ENV=development
```

Comment:
is this required? I think we are checking if environment is equal to production or not. in that case it should work if the production environments have the variable set

Reply:
Not required but either not a problem I think.  Cause I saw this for other microservices like search, gateway.